youtube_dl_server/server.py

```diff
@@ -29,6 +29,14 @@ def main():
         help='The host the server will use. The default is: %(default)s',
     )
 
+    parser.add_argument(
+        '--number-processes',
+        default=5,
```

Comment:
I'm not opposed, but maybe is better to use a value depending on the number of cores. See for example the [documentation for gunicorn](http://docs.gunicorn.org/en/latest/settings.html#workers) (they default to `1`), in one of the examples they use `2 x NCORES + 1`.